saleor/graphql/csv/types.py

```diff
@@ -0,0 +1,39 @@
+import graphene
+from graphene import relay
+from graphene_federation import key
+from graphql_jwt.exceptions import PermissionDenied
+
+from ...core.permissions import AccountPermissions
+from ...csv import models
+from ..core.connection import CountableDjangoObjectType
+from .enums import JobStatusEnum
+
+
+@key(fields="id")
+class Job(CountableDjangoObjectType):
+    url = graphene.String(description="The URL of field to download.")
+    status = JobStatusEnum(description="Job status.")
+
+    class Meta:
+        description = "Represents job data"
+        interfaces = [relay.Node]
+        model = models.Job
+        only_fields = ["id", "created_at", "ended_at", "status", "url", "user"]
+
+    @staticmethod
+    def resolve_url(root: models.Job, _info):
```

Comment:
`_blah` means unused `blah`
```suggestion
    def resolve_url(root: models.Job, info):
```